bitnami/influxdb/README.md

```diff
@@ -185,327 +169,273 @@ There are K8s distribution, such as OpenShift, where you can dynamically define
+| `defaultInitContainers.volumePermissions.resources`                                         | Set InfluxDB(TM) Core "volume-permissions" init-container requests and limits for different resources like CPU or memory (essential for production workloads)                                                                                                                                                                          | `{}`                       |
+
+### Other Parameters
+
+| Name                                          | Description                                                                                | Value   |
+| --------------------------------------------- | ------------------------------------------------------------------------------------------ | ------- |
+| `serviceAccount.create`                       | Enable creation of ServiceAccount for InfluxDB(TM) Core pods                               | `true`  |
+| `serviceAccount.name`                         | Name of the service account to use. If not set and `create` is `true`, a name is generated | `""`    |
+| `serviceAccount.automountServiceAccountToken` | Allows auto mount of ServiceAccountToken on the serviceAccount created                     | `false` |
+| `serviceAccount.annotations`                  | Additional custom annotations for the ServiceAccount                                       | `{}`    |
+
+Specify each parameter using the `--set key=value[,key=value]` argument to `helm install`. For example,
+
+```console
+helm install my-release \
+  --set objectStore=file \
+    oci://REGISTRY_NAME/REPOSITORY_NAME/influxdb
+```
+
+> Note: You need to substitute the placeholders `REGISTRY_NAME` and `REPOSITORY_NAME` with a reference to your Helm chart registry and repository. For example, in the case of Bitnami, you need to use `REGISTRY_NAME=registry-1.docker.io` and `REPOSITORY_NAME=bitnamicharts`.
+
+The above command sets the InfluxDB&trade; Core object store to `file`. Alternatively, a YAML file that specifies the values for the above parameters can be provided while installing the chart. For example,
+
+```console
+helm install my-release -f values.yaml oci://REGISTRY_NAME/REPOSITORY_NAME/influxdb
+```
+
+> Note: You need to substitute the placeholders `REGISTRY_NAME` and `REPOSITORY_NAME` with a reference to your Helm chart registry and repository. For example, in the case of Bitnami, you need to use `REGISTRY_NAME=registry-1.docker.io` and `REPOSITORY_NAME=bitnamicharts`.
+> **Tip**: You can use the default [values.yaml](https://github.com/bitnami/charts/tree/main/bitnami/influxdb/values.yaml)
 
 ## Troubleshooting
 
 Find more information about how to deal with common errors related to Bitnami's Helm charts in [this troubleshooting guide](https://docs.bitnami.com/general/how-to/troubleshoot-helm-chart-issues).
 
 ## Upgrading
 
+### To 7.0.0
```

Comment:
I see no mention to the changes in the admin token management

Reply:
addressed at https://github.com/bitnami/charts/pull/33743/commits/859c509824da5be460e8f02a9701f49e440ae834